database_cleanup/models/purge_modules.py

```diff
@@ -63,16 +63,19 @@ class CleanupPurgeWizardModule(models.TransientModel):
     @api.model
     def find(self):
         res = []
+        purge_lines = self.env['cleanup.purge.line.module']
```

Comment:
Is it worth adding `.browse()` for clarity (empty recordset)?
:+1: I have experienced slow processes when cleaning databases, maybe it was the cause.

Reply:
I used to add the `.browse([])`, but meanwhile I see this used as idiom for an empty recordset very often, so if you don't insist, I'd like to keep it this way

Reply:
I don't, it was a coding-style question!